.environment/ktlint/run-ktlintCheck.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+KTLINT_IMAGE_NAME="reportstream-ktlint:latest"
+REPO_ROOT=$(git rev-parse --show-toplevel)
+CONTAINER_SOURCE_LOCATION="/src"
+LOG_FILE="ktlint.log"
+
+function usage() {
+    echo "usage: ${0} [OPTION]"
+    echo ""
+    echo "Checks your code for ktlint violations."
+    echo "By default, this script will only check files that you added or changed (staged and unstaged)."
+    echo ""
+    echo "Options:"
+    echo "    --full|-f     Scans all files (shorthand: lower-case f)"
+    echo "    --format|-F   Fix violations (shorthand: UPPER-case F)"
```

Comment:
Having both options be "f" names is confusing. Could the full option be renamed to `--all` or `-a`?

Reply:
Good observation. I changed it to --all and -a